WooCommerce/WooCommerceUITests/Tests/ProductsTests.swift

```diff
@@ -3,6 +3,14 @@ import XCTest
 
 final class ProductsTests: XCTestCase {
 
+    let productTypes = [
+        "physical": "Simple physical product",
+        "virtual": "Virtual physical product",
```

Comment:
I think it should say

```suggestion
        "virtual": "Simple virtual product",
```

![SCR-20221212-rfw](https://user-images.githubusercontent.com/73365754/207120236-9cc4f8f7-3882-4326-b9ca-45ddce462a94.png)

Reply:
oof - such a bad typo, thanks for catching this! will fix it

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/8380/commits/91b15ef1f55c562f15434b7d1f7e8f3459308652#diff-ba22d4d1543946cc905fc3a52bdd2c2533cb2475eb78c9fb1e9bef76c9870fb9